### ISSUE TITLE
ReferenceError:“TextEncoder”未定义

### ISSUE THREAD

**Issue by wjyzhixing** [soldair/node-qrcode#373]
Hello!My node version is v16.20.0. 

I installed version 1.5.4 of qrcode, and I encountered an error on IE11. The error message is as follows:"

![1724416648537](https://github.com/user-attachments/assets/ccc3adc8-8556-43dc-bb98-c6b2218cd62d)

How can I deal with this problem?Or is this a bug?It will appreciate that if you can give me some advice.

**Comment by M393** [comment]
TextEncoder usage was added with #317 v1.5.4, so either use 1.5.3 or add a polyfill for TextEncoder like https://github.com/anonyco/FastestSmallestTextEncoderDecoder

**Comment by wjyzhixing** [comment]
yes,now I use v1.5.3 version.But if I use v1.5.4.If I alse use fastestsmallesttextencoderdecoder-encodeinto.I can only resolve the import of TextEncoder in my code.How to deal with the code in node_modules lib?my npm run build is base in babel.Here is my babel.config.js
![image](https://github.com/user-attachments/assets/8ba19350-3aa9-4d8b-8c67-99462398b6f5)
How can I do more to fix in v1.5.4?I realy want to learn this method to solve it.Thank you very much.

**Comment by hsuna** [comment]
you can use version 1.5.3

https://github.com/soldair/node-qrcode/commit/e09b913a65fc63da049a5577e5ac0eca22abac85
